### PR TITLE
[MOSIP-29675]Update administration.md

### PR DESCRIPTION
**Pull request by Mahesh-Binayak**

### CODE REVIEW

File: docs/administration.md [hunk 1]
```diff
@@ -23,10 +23,10 @@ The admin module provides the following functions:
 ## Activate/deactivate/decommission resources
 
 ### What is deactivation of a resource?
-Deactivation refers to a temporary shutdown. This means that the resource will not be available for use unless and until it is activated later through the admin portal as required by the country.
+Deactivation refers to a reversible action in which the isActive value for a resouce is database is set to "False". This means that the resource will not be available for use unless and until it is activated later through the admin portal as required by the country.
```
Comment (ghost) [inline docs/administration.md]: spell check "resource" and correct the sentence.

File: docs/administration.md [hunk 2]
```diff
@@ -23,10 +23,10 @@ The admin module provides the following functions:
...
 
 ### What is decommissioning a resource?
-* Decommission refers to a permanent shut down of the resource. This also automatically deactivates it. 
+* Decommission refers to a permanent/irreversible action of the resource. This also automatically deactivates it and the isDeleted value for the resource is set to "True".
```
Comment (ghost) [inline docs/administration.md:29]: irreversible action on the resource.

File: docs/administration.md (hunk 1 above)
Comment (Mahesh-Binayak) [reply docs/administration.md]: Thanks @anusha , corrected the spelling, pls approve/merge

**Review by Balakuntalam — APPROVED** [review]
I see minor (2 nos) of change and are OK. Hence, approved for merge.

**Pull request merged**